# S1

Esta libreta es para ajustar los valores y volumenes para la simulación de transporte de calor en Avizo. Lo que corresponde es agrupar todos los datos de los diferentes análisis y caracterizaciones para la determinación de volúmenes y conductividades térmicas para cada una de las fases.

___

## Estudios

### `MicroCT`
1. La imagen cruda con dimensiones de `994x1024x2259` con un tamaño de voxel de `28.7855 µm` se recortó para poder procesarla sin la presencia de artefactos, con los siguientes parámetros de recorte, resultando en una imagen de dimensiones `580x580x1776`
    - x = [218-797]
    - y = [216-795]
    - z = [229-2006]

2. El filtrado de la imagen se realizó con el filtro Generic-Denoising_v-1.0 de Dragonfly, un modelo de tipo regresión basado en una arquitectura U-Net (CNN). Este modelo cuenta con una profundidad de 5 niveles y un número inicial de 64 filtros, operando sobre imágenes bidimensionales (2D). Está diseñado para tareas de eliminación de ruido, preservando los detalles estructurales de la imagen mediante conexiones de salto características de la arquitectura U-Net.
3. La segmentación de la imagen recortada y filtrada, se realizó mediante una umbralización múltiple con los siguientes cortes:
    - Fase 1: [ 7,187 - 19,059]
    - Fase 2: [19,059 - 23,918]
    - Fase 3: [23,918 - 27,914]
    - Fase 3: [27,914 - 54,860]

4. El siguiente paso fue la identificación y cuantificación de las fases:
    - Matriz : 94.87%
    - Poros : 4.89%
    - Mineral 1 : 0.16%
    - Mineral 2 : 0.08%

___

### `Petrografía`
Con el análisis petrográficos de las láminas delgadas del tope y base del tapón de núcleo se concluyó con la siguiente interpretación
- Muestra: HJ-14-03-T01, HJ-14-03-B01
- Unidad: Qta3 Traquiandesitas San Antonio - Las Chapas
- Localidad: San Antonio, Chignautla, Puebla

Descripción microscópica:
- Textura: pilotaxítica o hialofítica

Mineralogía: 
- Primarios: Plagioclasa cálcica, opacos, piroxenos
- Secundarios: Opacos

Composición modal estimada: Plagioclasa > Opacos > Piroxenos

Clasificación de la roca: roca ígnea volcánica tipo lava

Comentarios adicionales: contiene vidrio

___

### `Elementos mayores`
| Estudio  | Muesta   | SiO<sub>2</sub> | TiO<sub>2</sub> | Al<sub>2</sub>O<sub>3</sub> | Fe<sub>2</sub>O<sub>3</sub> | MnO | MgO  | CaO  | Na<sub>2</sub>O | K<sub>2</sub>O | P<sub>2</sub>O<sub>5</sub> | LOI  | Total |
|----------|----------|-------|------|-------|-------|------|------|------|------|------|------|------|--------|
| Carrasco-Nuñez et al. (2017) | HJ-14-03 | 61.16 | 1.20 | 17.19 | 5.99  | 0.12 | 1.90 | 4.11 | 5.25 | 2.52 | 0.45 | 0.16 | 100.05 |
| FRX en CFATA                 | HJ-14-03 | 65.08 | 1.42 | 19.19 | 7.21  | 0.15 |   -  | 4.41 |   -  | 1.78 | 0.58 |   -  | 99.82  |
| diff                         |          | -3.92 | -0.22| -2.00 | -1.22 | -0.03| 1.90 | -0.30| 5.25 | 0.74 | -0.13| 0.16 | 0.23   |

___

### `DRX`
|   Fase      |    Fórmula química    |     PDF#    | Wt % |
|-------------|-----------------------|-------------|------|
|   Albita    |       NaAlSi₃O₈       | 04-007-5092 | 66.7 |
|  Anorthita  |       CaAl₂Si₂O₈      | 04-011-1371 | 18.3 |
|  Pigeonita  | Ca₀.₀₆Mg₀.₆Fe₀.₃₄SiO₃ | 04-017-7892 | 11.6 |
|  Magnetita  |   Fe(Fe₁.₂₄Ti₀.₆₁)O₄  | 04-017-7892 | 11.6 |

___

### `Comentarios`
En la vista del microscopio se me sugurió que el piroxeno es una augita, y no pigeonita

___

### `Resumen`
La siguiente tabla tiene las fases encontradas y demas propiedades que utilizadas en la simulacion
| Fase           | $\rho$ [g/cm³]| CT [W/m·K]   | Fórmula química           | Fase μCT | Fase (petrografía)| %DRX | %μCT |
|----------------|---------------|--------------|---------------------------|----------|-------------------|------|------|
| Ilmenita       | 4.5 – 5.0     | 2.5 – 3.5    | FeTiO₃                    | mineral2 | opaco             | 3.4  |      |
| Hematita       | 5.0 – 5.3     | 10 – 12      | Fe₂O₃                     | mineral2 | opaco             |  ''  |      |
| Magnetita      | 5.1 – 5.2     | 5 – 6        | Fe₃O₄                     | matriz   | opaco             |  ''  |      |
| Titanomagnetita| 4.5 – 5.2     | 2.5          | (Fe²⁺,Ti⁴⁺)(Fe³⁺)₂O₄      | matriz   | opaco             |  ''  |      |
| Albita         | 2.60 – 2.65   | 2.0 – 2.5    | NaAlSi₃O₈                 | matriz   | pl sodica         | 66.7 |      |
| Anortita       | 2.75 – 2.80   | 1.5 – 2.5    | CaAl₂Si₂O₈                | mineral1 | pl calcica        | 18.3 |      |
| Pigeonita      | 3.2 – 3.6     | 2.0 – 3.0    | (Ca,Mg,Fe)(Mg,Fe)Si₂O₆    | matriz   | piroxeno          | 11.6 |      |
| Poros          | 0.0012        | 0.026        | 78%N₂ + 21%O₂ + 1%otros   | poros    | poros             | -    | 4.89 |


___

## Cálculo de CT

### `Correspondencia de fases`
| Fase en MicroCT  | Rango de umbral     | % Vol. imagen  | Descripción                                  | Minerales principales                  |
|------------------|---------------------|----------------|----------------------------------------------|----------------------------------------|
| Fase 1           | 7,187 – 19,059      | 4.89%          | Poros                                        | Aire                                   |
| Fase 2           | 19,059 – 23,918     | 94.87%         | Matriz dominante                             | Albita, pigeonita, anortita            |
| Fase 3           | 23,918 – 27,914     | 0.16%          | Cristales (pl cálcica) individuales visibles | Anortita                               |
| Fase 4           | 27,914 – 54,860     | 0.08%          | Inclusiones densas y opacas                  | Magnetita, hematita, ilmenita          |

### `Cálculo de ETC para la fase 1`

Identificada en la microCT como poros asumimos que esta fase es aire. 
$$k^{\text{Fase 1}} = 0.026 \text{ W/m} \cdot \text{K}$$

### `Cálculo de ETC para la fase 2`

|   Mineral   |    Conductividad térmica (W/m·K)    | % frac |
|-------------|:-----------------------------------:|--------|
|   Albita    |                2.2                  | 0.667  |
|   Anortita  |                2.0                  | 0.183  |
|   Augita    |                2.5                  | 0.116  |

#### Límite de Voigt
$$k_V=\sum_{i}f_i \cdot k_i$$

In [7]:
kV = (0.667*2.2) + (0.183*2.0) + (0.116*2.5) 
kV

2.1234

#### Límite de Reuss
$$\frac{1}{k_R}=\sum_{i} \frac{f_i}{k_i}$$

In [8]:
kR = ((0.667/2.2) + (0.183/2.0) + (0.116/2.5) )**(-1)
kR

2.267153074053464

#### Límite medio efectivo estimado
$$k_{ETC} = \frac{k_V + k_R}{2}$$

In [10]:
kEff = (kV+kR)/2
kEff

2.195276537026732

$$k_{ETC}^{\text{Fase 2}} = \frac{k_V + k_R}{2} = 2.195 \text{ W/m} \cdot \text{K}$$

___

### `Cálculo de ETC para la fase 3`

Identificada en el microscopio como mineral dominante Anortita (pl cálcica), asumimos que esta fase visible es puramente anortita. 
$$k^{\text{Fase 2}} = 2.0 \text{ W/m} \cdot \text{K}$$

___

### `Cálculo de ETC para la fase 4`

|   Mineral        |    Conductividad térmica (W/m·K)    | % frac |
|------------------|:-----------------------------------:|--------|
|  Titanomagnetita |                2.5                  | 0.5    |
|     Ilmenita     |                3.0                  | 0.5    |

#### Límite de Voigt
$$k_V=\sum_{i}f_i \cdot k_i$$

In [11]:
kV = 0.5*2.5 + 0.5*3.0
kV

2.75

#### Límite de Reuss
$$\frac{1}{k_R}=\sum_{i} \frac{f_i}{k_i}$$

In [12]:
kR = (0.5/2.5 + 0.5/3.0)**(-1)
kR

2.727272727272727

#### Límite medio efectivo estimado
$$k_{ETC} = \frac{k_V + k_R}{2}$$

In [13]:
kEff = (kV+kR)/2
kEff

2.7386363636363633

$$k_{ETC}^{\text{Fase 4}} = 2.739 $$

___

### `Propiedades físicas asignadas`
| Fase         | CT (k, W/m·K) | Fase en imagen | Observaciones                             |
|--------------|----------------------------------|----------------|-------------------------------------------|
| Poros        | 0.026                            | Fase 1         | Aire                                      |
| Matriz       | 2.195                            | Fase 2         | Mezcla de albita + pigeonita + anortita   |
| Mineral 1    | 2.0                              | Fase 3         | Anortita (plagioclasa cálcica)            |
| Mineral 2    | 2.739                            | Fase 4         | Titanomagnetita / ilmenita (opacos)       |
